### ISSUE TITLE
Compare Multivar PSD LnL to orignal LnL 

### ISSUE THREAD

**Issue by avivajpeyi** [nz-gravity/FlowSGVB#2]
When no correlated noise the multivariate PSD lnl and the original LnL should be the same... Are they?

**Comment by jianan-joseph-liu** [comment]
Yes, the lihelihood for these two cases should be the same. The following plot show the values of likelihood given different values of M_c for two cases.
[likelihood_comparison.pdf](https://github.com/user-attachments/files/18111549/likelihood_comparison.pdf)
